kuma/wiki/templates/wiki/revision.html

```diff
@@ -52,6 +52,11 @@
       </div>
     </details>
 
+    <div class="wiki-block contributors">
+      {% set tags = document.tags.all() %}
+      {% include "wiki/includes/document_tag.html" with context %}
```

Comment:
`with context` is the default behavior for `include`s, so we typically leave it off our `include` tags.